config/compiler/BUILD.gn

```diff
@@ -1781,7 +1781,7 @@ config("no_size_t_to_int_warning") {
 # Code that currently generates warnings for this can include this
 # config to disable them.
 config("no_shorten_64_warnings") {
-  if (current_cpu == "x64") {
+  if (current_cpu == "x64" || current_cpu == "arm" || current_cpu == "arm64") {
```

Comment:
Are you sure about the "arm" -- seems to me this should only be arm64.

Reply:
It was added because of compiler warning 4267 (conversion from 'size_t' to 'type', possible loss of data). Warning is generated for arm and arm64 builds, so it should be added. It could be put in another config.

Reply:
Yes but does it apply to arm? seems to me this is only arm64, no?